File: loianeTraining/javaBasico/exercicios-aulas-14-15/Ex16.java
import java.util.Scanner;
import java.lang.Math;

class Ex16 {
	public static void main(String[] args) {
		Scanner scan = new Scanner(System.in);

		System.out.println("Digite o as variaveis A, B e C para calcular um equacao quadrada.");
		double a = scan.nextDouble();
		if (a == 0) {
			System.out.println("A equacao nao e de segundo grau");
			scan.close();
			return;
		}
		
		double b = scan.nextDouble();
		double c = scan.nextDouble();

		double delta = Math.pow(b, 2) - (4 * a * c);
		System.out.println("Delta = " + delta);

		if (delta < 0) {
			System.out.println("A equacao nao possui raizes reais");
		} else if (delta == 0) {
			double x1 = (-b + Math.sqrt(delta)) / (2 * a);
			System.out.println("A equacao possui apenas uma raiz real " + x1);
		} else {
			double x1 = (-b + Math.sqrt(delta)) / (2 * a);
			double x2 = (-b - Math.sqrt(delta)) / (2 * a);
			System.out.println("A equacao possui 2 raizes reais " + x1 + " | " + x2);
		}
		scan.close();
	}
}

File: loiane-training-csharp/procedural-poo/c/Ex16.cs
using System;
using System.Threading;

public class Ex16
{
    static void Main(string[] args)
    {
        Console.WriteLine("Digite o valor de A");
        double a = double.Parse(Console.ReadLine());
        Console.WriteLine("Digite o valor de B");
        double b = double.Parse(Console.ReadLine());
        Console.WriteLine("Digite o valor de C");
        double c = double.Parse(Console.ReadLine());

        if (a == 0)
        {
            return;
        }

        double Delta = Math.Pow(b, 2) - 4 * a * c;
        Console.WriteLine("Delta: " + Delta);

        if (Delta < 0) {
            Console.WriteLine("Equacao nao possui raizes reais!");
        } else if (Delta == 0) {
            double Raiz = -b * Math.Sqrt(Delta) / 2 * a;
            Console.WriteLine("Raiz: " + Raiz);
        } else if (Delta > 0) {
            double RaizX = -b + Math.Sqrt(Delta) / 2 * a;
            double RaizY = -b - Math.Sqrt(Delta) / 2 * a;
            Console.WriteLine("Raiz X: " + RaizX + "\nRaiz Y: " + RaizY);
        }

        Thread.Sleep(5000);
    }
}
